src/handlers/comment/handlers/query.ts

```diff
@@ -28,12 +28,18 @@ export const query = async (body: string) => {
     let data = await getAllAccessLevels(user, repo.full_name);
     if (!data) {
       logger.info(`Access info does not exist for @${user}`);
-      data = {
-        multiplier: false,
-        priority: false,
-        time: true,
-        price: false,
-      };
+      try {
+        await upsertAccessControl(user, repo.full_name, "time_access", true);
+        data = {
+          multiplier: false,
+          priority: false,
+          time: true,
+          price: false,
+        };
+      } catch (e) {
+        logger.error(`Exception thrown by getAllAcessLevels: ${e}`);
```

Comment:
I thought we are using a new error function so that the bot posts it in diff format @Keyrxng rfc